src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -298,6 +298,18 @@
       </BuildCompleteTemplateV2>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(HelixCreator)' != ''">
```

Comment:
I don't care strongly but you could probably make this work with 1 ItemGroup and the condition on the item.